package-diff

```diff
@@ -55,9 +55,21 @@ if [ "$FILE" = flatcar_production_image_contents.txt ] || [ "$FILE" = flatcar_de
   fi
   if [ "$CALCSIZE" = 1 ]; then
     A_SUM=$(($(grep -v '^d' "$A" | grep -v '^l' | tr -c '[:graph:][:space:]' '?' | rev | cut -d ' ' -f 2 | rev | paste -sd+ -)))
-    echo "$((${A_SUM}/1024/1024)) MiB" > "$A"
+    A_BOOT=$(($(grep -v '^d' "$A" | grep -v '^l' | grep " [\.]*/boot" | tr -c '[:graph:][:space:]' '?' | rev | cut -d ' ' -f 2 | rev | paste -sd+ -)))
+    A_USR=$(($(grep -v '^d' "$A" | grep -v '^l' | grep " [\.]*/usr" | tr -c '[:graph:][:space:]' '?' | rev | cut -d ' ' -f 2 | rev | paste -sd+ -)))
+    A_ROOT=$(($(grep -v '^d' "$A" | grep -v '^l' | grep -v " [\.]*/usr" | grep -v " [\.]*/boot" | tr -c '[:graph:][:space:]' '?' | rev | cut -d ' ' -f 2 | rev | paste -sd+ -)))
```

Comment:
[insert a gif of me putting cake mixer in my eyeballs]

Looks good!

Reply:
I think it can be simplified but at the moment I don't really want to invest more time

Reply:
Follow up PRs welcome ;)

Reply:
Sure, maybe split out some code to a function or something, but eh - there are more pressing things to do.